mobile/library/common/internal_engine.cc

```diff
@@ -11,6 +11,9 @@
 #include "library/common/stats/utility.h"
 
 namespace Envoy {
+namespace {
+constexpr absl::Duration ENGINE_RUNNING_TIMEOUT = absl::Seconds(3);
```

Comment:
out of curiosity, any idea how long average startup time is?  I'm not sure what the typical range is

Reply:
It's typically really fast (less than 1 second). I put in 3 seconds and it's pretty arbitrary but I don't want it to wait indefinitely. I'm open to changing it to whatever timeout value that you think is more reasonable.

Reply:
3s SGTM